ServerCore/Pages/Hints/Create.cshtml

```diff
@@ -2,10 +2,14 @@
 @model ServerCore.Pages.Hints.CreateModel
 
 @{
-    ViewData["Title"] = "Create";
+    ViewData["Title"] = "Create hint";
+    //Needs route data - ViewData["AdminRoute"] = "/Hints/Create";
```

Comment:
Maybe go somewhere like the puzzles list in these cases? Not sure.

Reply:
I'm hoping I can figure this out soon enough that we won't have to deal with it for long - if I can't get it working I can look to switch to a different backup (right now they all go to the event home page